packages/@azure/servicebus/data-plane/lib/connectionContext.ts

```diff
@@ -13,7 +13,7 @@ import {
   delay,
   TokenProvider
 } from "@azure/amqp-common";
-import { NamespaceOptions } from "./namespace";
+import { ServiceBusClientOptions } from "./namespace";
```

Comment:
should we also rename the file from `namespace` to `servicebusclient`?

Reply:
Done